wsba-participant-completion-simple/src/test/java/org/jboss/as/quickstarts/wsba/participantcompletion/simple/Client.java

```diff
@@ -57,6 +58,7 @@ public Client() throws MalformedURLException {
          * Add client handler chain so that XTS can add the transaction context to the SOAP messages.
          */
         BindingProvider bindingProvider = (BindingProvider) set;
+        @SuppressWarnings("rawtypes")
```

Comment:
I tried to fix this, so that the warning is never raised. However, setHandlerChain on line 64 doesn't seem to accept generic types. I'm not that hot on generics, but i think it could be fixed with some casting on line64, but it's probably not worth it.